lms/djangoapps/edxnotes/views.py

```diff
@@ -205,3 +206,13 @@ def edxnotes_visibility(request, course_id):
             "Could not decode request body as JSON and find a boolean visibility field: '%s'", request.body
         )
         return JsonResponseBadRequest()
+
+
+@login_required
+def edxnotes_delete_user_all(request, user_id):
+    """
+    LMS endpoint for edxnotes - no URL pattern currently enabled
+    """
+    if not request.user.is_staff:
```

Comment:
I'm pretty sure it's supposed to be superusers only, plus one specific account for the Jenkins user. Might want to check in with the platform team on that though.

Reply:
@sanfordstudent No doubt. Will follow up.